resources/js/Components/NavBar/NavBar.vue

```diff
@@ -47,15 +51,15 @@
             </a>
         </div>
         <div class="navbar-center">
-            <img class="block lg:hidden h-8 w-auto"  :src="'images/logo-mini-blue.svg'" /> 
+            <img class="block lg:hidden h-8 w-auto"  :src="'images/logo-mini-blue.svg'" />
```

Comment:
Aqui podemos passar diretamente também:

```suggestion
            <img class="block lg:hidden h-8 w-auto"  src="images/logo-mini-blue.svg" />
```